### ISSUE TITLE
go build error on linux

### ISSUE THREAD

**Issue by qetee** [otiai10/gosseract#61]
when i use `gox -osarch="darwin/amd64" to build; it shows:

```
1 errors occurred:
--> darwin/amd64 error: exit status 1
Stderr: ../vendor/github.com/otiai10/gosseract/goss.go:3:8: no buildable Go source files in /home/viggo/Documents/GoPath/src/vend
or/github.com/otiai10/gosseract/tesseract
```

but it's ok when i use `gox -osarch="linux/amd64"`

And my system is debian 8

**Comment by zouhuigang** [comment]
你这个是怎么解决的？难道不能交叉编译？必须设置成"linux/amd64"?

**Comment by otiai10** [comment]
@qetee 
Hi, sorry for being late.
I guess gosseract should be managed by `go get`, not `dep`, because dep doesn't care about directories which don't have any go source. (is it right? I'm not sure).

I'm rebuilding this repository with replacing by pure go source, in this [milestone](https://github.com/otiai10/gosseract/milestone/3). I hope this issue would be solved on `v2.0`
